media/vscode.css

```diff
@@ -61,6 +61,7 @@ button {
   outline-offset: 2px !important;
   color: var(--vscode-button-foreground);
   background: var(--vscode-button-background);
+  margin: 2.5px 2.5px 2.5px 2.5px;
 }
```

Comment:
small margin around the recipe buttons

Reply:
**todo:** this has very slightly messed up the layout, I think
![Screenshot from 2023-06-09 10-45-59](https://github.com/sourcery-ai/sourcery-vscode/assets/15631502/75747c16-06ca-4034-bc26-5c3b88b1181d)
The buttons are now a little wider than the box, so there's a horizontal scroll bar, and that makes them clipped on the right hand side.